playwright/accessibility/Icon_Android.spec.mjs

```diff
@@ -0,0 +1,8 @@
+// @flow strict
+import { test } from '@playwright/test';
+import expectAccessiblePage from './expectAccessiblePage.mjs';
+
+test('Android Icon Accessibility check', async ({ page }) => {
+  await page.goto('/android/icon');
```

Comment:
@khalvy - looks like the accessibility check for the android icon/check failed.

If you click this button, you can see the error:
<img width="214" alt="image" src="https://user-images.githubusercontent.com/5509813/186774460-f17ddd1f-23dc-4ada-a0f5-d8084b49a168.png">

It seems like the images are missing alt-tags